packages/gatsby/src/commands/build-html.ts

```diff
@@ -24,6 +24,11 @@ import { processNodeManifests } from "../utils/node-manifest"
 import type { GatsbyWorkerPool } from "../utils/worker/pool"
 type IActivity = any // TODO
 
+const isPreview =
+  typeof process.env.GATSBY_IS_PREVIEW === `boolean`
```

Comment:
I think `process.env` can't ever be of type boolean? It is always converted to string. Or am I missing something? Did a quick check:

```js
process.env.GATSBY_IS_PREVIEW = true
console.log(typeof process.env.GATSBY_IS_PREVIEW) // returns "string" (value is "true")
```

Reply:
oh huh right 🤦